src/routes/api/rooms/queryRooms.ts

```diff
@@ -8,12 +9,31 @@ const queryRoomsRoute = async (req: Request, res: Response): Promise<void> => {
             res.status(404).json({ error });
             return;
         });
-        res.status(200).json({ room });
+        if (room) {
```

Comment:
This is just a temporary query of sections I put here. This will be addressed in another pr where i will add the query of sections from room models